README.md

```diff
@@ -1,81 +1,94 @@
-| Akka support | out of the box | requires custom serializer | requires custom serializer | requires custom serializer | out of the box |
-| Runtime safety | none <ul><li>uses reflection</li><li>errors appear only in runtime</li></ul> | encoders and decoders are checked during compile time | generates scala classes | standard for scala code | encoders and decoders are checked during compile time
-| Schema evolution | <ul><li>removing field</li><li>adding optional field</li></ul> with [`JacksonMigration`](https://doc.akka.io/docs/akka/current/serialization-jackson.html#schema-evolution) <ul><li>adding mandatory field</li><li>renaming field</li><li>renaming class</li><li>support of forward versioning for rolling updates</li></ul>| <ul><li>adding optional field</li><li>removing optional field</li><li>adding required field with default value</li><li>removing required field</li><li>renaming class</li></ul> | <ul><li>switching between optional and repeated field</li><li>adding new fields</li><li>renaming fields</li></ul> | <ul><li>reordering fields</li><li>renaming fields</li><li>adding optional field</li><li>adding required field with default value</li><li>removing field with default value</li></ul> | any arbitrary transformation can be defined manualy with transcoders
-| Boilerplate | a lot: <ul><li>ADTs requires amount of annotation equal to or exceeding the actual type definitions</li><li>poor support for case object - needs sealed trait per case object and a custom deserializer</ul> | none, Circe can derive codecs from case class definitions | in case of custom types a second layer of models is needed | sometimes requires annotations | considerable: every top level sealed trait must have manually defined codec
+| Serializer | Jackson | Circe | Protobuf v3 | Avro | Borer | Kryo
+|:---|:---|:---|:---|:---|:---|:---|
+| Data formats | JSON or [CBOR](https://cbor.io) | JSON | JSON or binary | JSON or binary | JSON or CBOR | binary
+| Scala support | with [`jackson-module-scala`](https://github.com/FasterXML/jackson-module-scala) <ul><li>lacks support of basic scala types like `Unit`</li><li>without explicit annotation doesn't work with generics extending `AnyVal`</ul> | perfect | with [`ScalaPB`](https://scalapb.github.io) generates Scala ADTs based on protobuf definitions | with [`Avro4s`](https://github.com/sksamuel/avro4s) library generaters Avro schema based on Scala ADTs | perfect | perfect
+| Akka support | [library](https://doc.akka.io/docs/akka/current/serialization-jackson.html) | requires custom serializer | [akka.remote](https://doc.akka.io/docs/akka/current/serialization.html) | requires custom serializer | requires custom serializer | [library](https://github.com/altoo-ag/akka-kryo-serialization)
+| Runtime safety | none, uses reflection | encoders and decoders are checked during compile time | standard for scala code | standard for scala code | encoders and decoders are checked during compile time | none, uses reflections
```

Comment:
Nit:
```suggestion
| Runtime safety | none, uses reflection | encoders and decoders are checked during compile time | standard for scala code | standard for scala code | encoders and decoders are checked during compile time | none, uses reflection
```

Reply:
Also... for Protobuf, where is that `akka.remote` name coming from? 🤔 akka-remote library itself is typically written with a hyphen

Reply:
Serializer comes from `akka-remote`

Reply:
Ok so pls change to `akka-remote` with hyphen then ➖